### PR TITLE
WIP: Error fixing

### PR DESCRIPTION
**Pull request by mhoeber**

### CODE REVIEW

**Comment by mhoeber** [comment]
@lamagnifica   this fixed some build errors and PDF issues:  http://draft-error-fixing.readthedocs.org/en/latest/

File: en_us/course_authors/source/content_experiments/content_experiments_add.rst [hunk 1]
```diff
@@ -34,9 +34,8 @@ To configure a content experiment in Studio, you:
 #. `Create the content experiment`_.
 #. `Create content for groups in the content experiment`_.
    
-After you configure a content experiment, you can `change the
```
Comment (lamagnifica) [inline en_us/course_authors/source/content_experiments/content_experiments_add.rst:37]: line 7 in this file still gives this error for a make html:

/Users/alisonhodges/edx-documentation/en_us/course_authors/source/content_experiments/content_experiments_add.rst:7: WARNING: undefined label: set up group configurations in an xml course (if the link has no caption the label must precede a section header)

Similar error for make latexpdf:

None:None: WARNING: undefined label: set up group configurations in an xml course (if the link has no caption the label must precede a section header)

File: en_us/course_authors/source/releasing_course/d.rst [hunk 2]
```diff
@@ -1,38 +0,0 @@
...
-
-    *Note Setting a grace period is generally not recommended. It can lead to problems not closing "when they should", and may be misleading to your students.*
-
-    If you have further questions about specifying times and time zones, or are experiencing inconsistencies in due date or release date behavior, please contact us from the edX Studio Help page.
-
-    **References**
-
-    http://help.edge.edx.org/discussions/questions/61-time-zones
-
-    http://help.edge.edx.org/discussions/questions/23-grace-periods
```
Comment (lamagnifica) [inline en_us/course_authors/source/releasing_course/d.rst:38]: Yay! thank you for deleting this file!

File: en_us/shared/exercises_tools/create_exercises_and_tools.rst [hunk 3]
```diff
@@ -181,7 +181,7 @@ STEM Exercises and Tools
           :alt: Example molecule editor problem
      - :ref:`Molecule Editor`
      - The molecule editor allows students to draw molecules that follow the rules for covalent bond formation and formal charge, even if the molecules are chemically impossible, are unstable, or do not exist in living systems.
-   * - .. image:: /Images/Molecule_Viewer.png
+   * - .. image:: /Images/MoleculeViewer.png
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/create_exercises_and_tools.rst:184]: omg, I can't believe we missed this undeerscore.
Anyway, sadly, NONE of these images files are getting picked up correctly in any of the HTML tables on this page. I checked docs.edx.org too -- no good there either.

**Comment by lamagnifica** [comment]
This will clear up a lot of pesky errors. HTML looks good. I do continue to fail when I build the PDF. The files run with the one warning ntoed above, then:
`build succeeded, 1 warning.`
`Running LaTeX files through pdflatex...`
`/Applications/Xcode.app/Contents/Developer/usr/bin/make -C build/latex all-pdf`
`pdflatex  'BuildingandRunninganedXCourse.tex'`
`make[1]: pdflatex: No such file or directory`
`make[1]: *** [BuildingandRunninganedXCourse.pdf] Error 1`
`make: *** [latexpdf] Error 2`
`Alisons-MacBook-Air:course_authors alisonhodges$`

File: en_us/course_authors/source/building_course/setting_up_student_view.rst [hunk 4]
```diff
@@ -110,9 +110,10 @@ The following example shows the course start date and time on the course summary
 .. image:: ../Images/about-page-course-start.png
  :alt: An image of the course About page, with the start date circled.
 
-.. note:: For courses on edX.org_, you must communicate the course start 
-  date and time to your edX program manager to ensure the date is accurate on
- the course About page.
+.. note:: 
+ For courses on edX.org_, you must communicate the course start date and time
+  to your edX program manager to ensure the date is accurate on the course
+  About page.
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/setting_up_student_view.rst]: in the HTML, the fist line under "note" is now bolded, and the second line is indented beneath it. I think adding a space before "For courses..." will fix both.

File: en_us/course_authors/source/content_experiments/content_experiments_add.rst [hunk 5]
```diff
@@ -34,9 +34,8 @@ To configure a content experiment in Studio, you:
...
-group configuration <Change the Group Configuration for a Content
-Experiment>`_.
+After you configure a content experiment, you can change its group
+configuration. See `Change the Group Configuration for a Content Experiment`_.
```
Comment (lamagnifica) [inline en_us/course_authors/source/content_experiments/content_experiments_add.rst:39]: this link ^ looks and works correctly now in the HTML.

**Comment by lamagnifica** [comment]
first pass complete

File: en_us/course_authors/source/content_experiments/content_experiments_add.rst (hunk 1 above)
Comment (mhoeber) [inline en_us/course_authors/source/content_experiments/content_experiments_add.rst:37]: removed link as this wasn't needed for olx sharing

File: en_us/course_authors/source/building_course/setting_up_student_view.rst (hunk 4 above)
Comment (mhoeber) [inline en_us/course_authors/source/building_course/setting_up_student_view.rst]: fixed

**Comment by mhoeber** [comment]
@lamagnifica   changes made, RTD draft project updated.

**Comment by lamagnifica** [comment]
:+1: :dancers:

**Pull request merged**